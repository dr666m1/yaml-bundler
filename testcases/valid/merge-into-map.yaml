foo: !include
  - filepath: ./sub/merge-into-map.yaml
    jsonpath: '$.one'
  - filepath: ./sub/merge-into-map.yaml
    jsonpath: '$.two'
