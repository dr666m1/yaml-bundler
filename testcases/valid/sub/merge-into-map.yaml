one:
  foo: bar
two:
  baz: qux
