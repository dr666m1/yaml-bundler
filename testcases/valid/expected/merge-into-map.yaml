foo:
  foo: bar
  baz: qux
